types/koji-tools/index.d.ts

```diff
@@ -0,0 +1,63 @@
+// Type definitions for koji-tools 0.5
+// Project: https://github.com/madewithkoji/koji-tools
+// Definitions by: Jeff Peterson <https://github.com/bdjeffyp>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
+
+// tslint:disable-next-line:export-just-namespace
+export = Koji;
```

Comment:
Do not disable this. Remove namespace declaration and export the declarations inside it directly.

Reply:
Done!